backend/core/src/main/java/com/feed_grabber/core/request/RequestRepository.java

```diff
@@ -24,5 +26,13 @@
 
     List<Request> findAllByQuestionnaireId(UUID id);
 
+    @Query("select r from Request r where r.closeDate is not NULL and r.questionnaire.company.id = :companyId")
+    List<Request> findAllReports(UUID companyId);
+
+    @Query("select r from Request r " +
+            "where r.closeDate is not null " +
+            "and r.sendToTarget = true and r.targetUser.id = :userId")
+    List<Request> findReportsForEmployee(UUID userId);
```

Comment:
Search available reports for simple user (employee).

Reply:
same with this method